sdk/keyvault/azure-security-keyvault-certificates/src/samples/java/com/azure/security/keyvault/certificates/BackupAndRestoreOperations.java

```diff
@@ -50,7 +50,7 @@ public static void main(String[] args) throws IOException, InterruptedException,
         //   already exists in the key vault, then a new version of the certificate is created.
         CertificatePolicy policy = new CertificatePolicy("Self", "CN=SelfSignedJavaPkcs12")
             .setSubjectAlternativeNames(new SubjectAlternativeNames().setEmails(Arrays.asList("wow@gmail.com")))
-            .setReuseKey(true)
+            .setKeyReusable(true)
```

Comment:
Should still be "ReuseKey" unless this is idiomatic, in which case shouldn't it have "is" or "can" as a prefix?

Reply:
is / can applies to only getters, 
setters always start with set prefix.
the variable name pattern should be same for both setter and getter i.e
is{varName}, set{varName}